### PR TITLE
Fix airgap for sonobuoy

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
Fix https://github.com/rancher/rancher/issues/26041

File: charts/rancher-cis-benchmark/v0.0.1/values.yaml [hunk 1]
```diff
@@ -22,9 +22,14 @@ debugWorker: false
...
-  tag: v0.1.9
-  pullPolicy: Always
+  securityScan:
+    repository: rancher/security-scan
+    tag: v0.1.9
+    pullPolicy: Always
+  sonobuoy:
+    repository: rancher/sonobuoy-sonobuoy
+    tag: v0.16.3 
+
```
Comment (prachidamle) [inline charts/rancher-cis-benchmark/v0.0.1/values.yaml:32]: do we need pullPolicy for sonobuoy?

File: charts/rancher-cis-benchmark/v0.0.1/values.yaml [hunk 2]
```diff
@@ -22,9 +22,14 @@ debugWorker: false
 debugTime: "infinity"
 
 image:
-  repository: rancher/security-scan
-  tag: v0.1.9
-  pullPolicy: Always
+  securityScan:
+    repository: rancher/security-scan
```
Comment (prachidamle) [inline charts/rancher-cis-benchmark/v0.0.1/values.yaml:26]: We need to change the tags in templates/pod.yaml as well for rancher/security-scan image

**Review by prachidamle — DISMISSED** [review]
comments inline

File: charts/rancher-cis-benchmark/v0.0.1/values.yaml (hunk 1 above)
Comment (StrongMonkey) [reply charts/rancher-cis-benchmark/v0.0.1/values.yaml:32]: We don't, it is not configurable

File: charts/rancher-cis-benchmark/v0.0.1/values.yaml (hunk 2 above)
Comment (StrongMonkey) [reply charts/rancher-cis-benchmark/v0.0.1/values.yaml:26]: Yeah, I changed the corresponding pod.yaml

Comment (deniseschannon) [reply charts/rancher-cis-benchmark/v0.0.1/values.yaml:26]: @StrongMonkey she means in here: https://github.com/rancher/system-charts/blob/dev/charts/rancher-cis-benchmark/v0.0.1/templates/pod.yaml#L68

as per her PR: https://github.com/rancher/system-charts/pull/183/files#diff-034d4bcb31f271b5aef68335ed19b67bL68

Comment (StrongMonkey) [reply charts/rancher-cis-benchmark/v0.0.1/values.yaml:26]: :+1: missed that

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**